Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/ApproveDeletionProcess/HandlerTests.cs

```diff
@@ -35,7 +37,9 @@ public async Task Happy_path()
         A.CallTo(() => mockIdentitiesRepository.FindByAddress(identity.Address, A<CancellationToken>._, A<bool>._))
             .Returns(identity);
 
-        var handler = CreateHandler(mockIdentitiesRepository, fakeUserContext);
+        var pushNotificationSender = A.Dummy<IPushNotificationSender>();
+
+        var handler = CreateHandler(mockIdentitiesRepository, fakeUserContext, pushNotificationSender: pushNotificationSender);
```

Comment:
Why the named parameter?

```suggestion
        var handler = CreateHandler(mockIdentitiesRepository, fakeUserContext, pushNotificationSender);
```

Reply:
I'm not passing the eventBus. Would you rather have the following?
```c#
var handler = CreateHandler(mockIdentitiesRepository, fakeUserContext, null,  pushNotificationSender);
```

Reply:
Create an overload without that parameter. Just like we do it in the other tests.